<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="407" started-at="2019-06-28T18:54:00Z" finished-at="2019-06-28T18:54:00Z">
    <groups>
    </groups>
    <test name="Login Test" duration-ms="407" started-at="2019-06-28T18:54:00Z" finished-at="2019-06-28T18:54:00Z">
      <class name="com.mj.testcases.UserLogin">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.mj.testcases.UserLogin@4cc8eb05]" name="setUp" is-config="true" duration-ms="12702" started-at="2019-06-29T00:23:47Z" finished-at="2019-06-29T00:24:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addCustomer(java.lang.String, java.lang.String)[pri:0, instance:com.mj.testcases.UserLogin@4cc8eb05]" name="addCustomer" duration-ms="0" started-at="2019-06-29T00:24:00Z" finished-at="2019-06-29T00:24:00Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [addCustomer] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [addCustomer] with [class java.lang.String, class java.lang.String].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:244)
	at org.testng.internal.Parameters.createParameters(Parameters.java:172)
	at org.testng.internal.Parameters.createParameters(Parameters.java:458)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:569)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.mj.testcases.UserLogin@4cc8eb05]" name="tearDown" is-config="true" duration-ms="1373" started-at="2019-06-29T00:24:00Z" finished-at="2019-06-29T00:24:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.mj.testcases.UserLogin -->
    </test> <!-- Login Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
